package.json

```diff
@@ -1,39 +1,44 @@
 {
-  "name": "goals-site",
+  "name": "goal-site",
   "version": "1.0.0",
   "private": true,
   "scripts": {
     "dev": "npm-run-all --parallel dev:*",
```

Comment:
![image](https://user-images.githubusercontent.com/57758447/185787551-ebef0c99-e0b1-4130-a866-80755bf041cb.png)

Yarn Dev Failed

I think it is because the packages are not being installed properly on running `yarn` command